lib/routes/discuz/discuz.ts

```diff
@@ -1,8 +1,7 @@
 import { Route } from '@/types';
 import cache from '@/utils/cache';
-import got from '@/utils/got';
+import ofetch from '@/utils/ofetch';
 import { load } from 'cheerio';
-import iconv from 'iconv-lite';
```

Comment:
You can't assume every site use uft-8. Doing so brings back #10143

Reply:
I missed checking the site that does not use UTF-8 because I only checked the second route in #10143. I will submit another commit with charset handling.